product-service/scripts/add_stocks.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO stock (product_id, count)
+VALUES ('0c43493a-7c7c-4894-8b08-3ecb453d5678', 2),
```

Comment:
How you could be sure, that products will have exactly these ids?

Reply:
It is just example. These ids are copied from existing products table after inserting the products into database.